# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/nsWindow.h
# Commit: a7b3d464b135
# Full Hash: a7b3d464b135d189f8dc61889a9f3199eb601169
# Author: Robert Mader <robert.mader@posteo.de>
# Date: 2021-04-27 22:18:30
# Regressor Bug: 1707577
# File Overlap Count: 1
# Description:
#   Bug 1707577 - Initialize variables in the constructor in cpp fashion, r=stransky
#   
#   Mostly in order to match code style.
#   Also add some missing variables and remove unused ones.
#   
# ==============================================================================

diff -r b5ba2f343559 -r a7b3d464b135 widget/gtk/nsWindow.h
--- a/widget/gtk/nsWindow.h	Tue Apr 27 18:57:58 2021 +0000
+++ b/widget/gtk/nsWindow.h	Tue Apr 27 18:57:59 2021 +0000
@@ -378,7 +378,6 @@
 
 #ifdef MOZ_X11
   Display* XDisplay() { return mXDisplay; }
-  Window mOldFocusWindow;
 #endif
 #ifdef MOZ_WAYLAND
   bool GetCSDDecorationOffset(int* aDx, int* aDy);
@@ -516,7 +515,7 @@
   GtkWidget* mShell;
   MozContainer* mContainer;
   GdkWindow* mGdkWindow;
-  bool mWindowShouldStartDragging = false;
+  bool mWindowShouldStartDragging;
   PlatformCompositorWidgetDelegate* mCompositorWidgetDelegate;
 
   uint32_t mHasMappedToplevel : 1, mRetryPointerGrab : 1;
